### PR TITLE
Add theming articles

### PR DESCRIPTION
**Pull request by kerpecheva**

### CODE REVIEW

File: styling-and-themes/customize-the-theme.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+There are several ways to customize the visual appearance of the Telerik .NET MAUI controls depending on the needed changes. You can modify only the colors in a certain color variation and that will affect all the controls or you can make more specific cbange in the styles and templates of a certain Telerik UI for .NET MAUI component.
+
+Either approach requires copying the needed resources in a separate resource dictionary, modifying it and merging it after the Telerik theme resources.
+
+Check the detailed steps below:
+
+### Copy and modify the theme resources
+
+Inside the **Telerik Theming** folder you have all the theme color variations (inside the **Swatches** folder) as well as the styles and templates of the Telerik .NET MAUI controls (inside the **Styles** folder).
```
Comment (yordan-mitev) [inline styling-and-themes/customize-the-theme.md:20]:
```suggestion
All available theme styles are in the `TelerikTheming` folder:
* The color variations are in the `Swatches` folder.
* The styles and templates for the Telerik UI for .NET MAUI controls are in the `Styles` folder.
```

**Review by yordan-mitev — COMMENTED** [review]
- The docs don't build locally because the new section hasn't been added to the `_config.yml`.
- As you add an entire section, consider moving the related styling articles and sections as child items, for instance, the Font Icons section.

File: styling-and-themes/overview.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### Theme
+
+A *theme* is a collection of styles and templates in XAML files, which determine the appearance of the Telerik .NET MAUI components, including fonts, colors, sizes and layouts.
+
+### Swatch
+
+A *theme swatch* is a color variation of a theme. All swatches of a given theme use the same fonts, sizes, and layouts. On the other hand, the text colors, background colors and border colors are different. For example, *Purple* and *Purple Dark* are two built-in swatch names.
+
+The Telerik .NET MAUI theme comes with a set of 8 predefined swatches for both dark and light modes: `Main`, `Main Dark`, `Ocean Blue`, `Ocean Blue Dark`, `Purple`, `Purple Dark`, `Turquoise` and `Turquoise Dark`.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply styling-and-themes/overview.md:22]: that's how they should be set in the code - with the empty space for the "Dark" options.

File: styling-and-themes/overview.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+```XAML
+<telerik:RadToggleButton x:Name="toggleButton"
+                         Content="Wi-Fi" />
+```
+
+The result is:
+
+![Telerik .NET MAUI ToggleButton with Telerik theme](images/togglebutton-themed.gif)
+
+>Keep in mind some of the Telerik UI for .NET MAUI controls do not fully support the Telerik Theming - AIPrompt, Chart, DataPager, ImageEditor, SlideView and Toolbar.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply styling-and-themes/overview.md:95]: Intentionally left it as a note, as there is some work done for these controls, just there are still issues left.

File: styling-and-themes/overview.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### Using the Theming Color System
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply styling-and-themes/overview.md:97]: Agree with the feedback, considering how and where to present this content

**Pull request merged**